server/server.js

```diff
@@ -11,6 +13,7 @@ app.use(bodyParser.json());
 
 app.use(express.static('public'));
 
+app.use(cors());
 
```

Comment:
This works for cors, but it opens it to all origins. The object in the function call parameter can set the origin(s) to specific URL(s). No need to change. Just thought you'd like to know.

Reply:
oh good point.  I can set the origin specifically to localhost instead of opening the entire flood gate.  Thanks for the tip!

Reply:
done.